packages/json-api/package.json

```diff
@@ -17,13 +17,13 @@
     "url": "git+https://github.com/surgeventures/noname.git"
   },
   "scripts": {
-    "prebuild": "rimraf ./dist",
     "build": "babel ./src -d ./dist --ignore ./src/**/*.test.js",
```

Comment:
We don't need babel if we are using typescript. We can compile directly to es5

Reply:
Fixed